### ISSUE TITLE
Document when babel-polyfill is needed

### ISSUE THREAD

**Issue by tschaub** [babel/babel#2773]
I'm hoping to get some clarity on when it is necessary to `require('babel-polyfill')`.

I notice that after the update to babel@6, the polyfill is required to use `instanceof` in browsers without `Symbol`.

For example:
```
$ echo 'new Date() instanceof Date' | babel --presets es2015
"use strict";

function _instanceof(left, right) { if (right != null && right[Symbol.hasInstance]) { return right[Symbol.hasInstance](left); } else { return left instanceof right; } }

_instanceof(new Date(), Date);
```

So it looks like using `instanceof` with the `es2015` preset requires `babel-polyfill` to use the generated code in an ES5 runtime.

Should I assume that using Babel to transpile ES2015 to ES5 *always* requires `babel-polyfill` now?

Thanks for any clarification.

**Comment by sebmck** [comment]
See the README:

> For questions and support please visit the <a href="https://babel-slack.herokuapp.com">Slack community</a> or <a href="http://stackoverflow.com/questions/tagged/babeljs">StackOverflow</a>.
>
> The Babel issue tracker is <strong>exclusively</strong> for bug reports and feature requests.
>
> For documentation and website issues please visit the <a href="https://github.com/babel/babel.github.io">babel.github.io</a> repo.

**Comment by tschaub** [comment]
Thanks for the pointer @sebmck.  I'll take this up in babel/babel.github.io#491 as I think that is essentially the same issue.

**Comment by tschaub** [comment]
Actually, it looks like a similar issue with `Symbol` was brought up in #2745  which was closed with 31a6cb9c5750253f30a6d3db6e441e5bdb879499.  Maybe this is more than a documentation issue.

**Comment by tschaub** [comment]
In case others find the same, #2784 should address the issue with `instanceof` requiring a `Symbol` polyfill.
